src/components/timeline/index.js

```diff
@@ -153,6 +161,49 @@ class TimelineSettingsHiddenTracks extends React.PureComponent<{|
   }
 }
 
+class TimelineSettingsActiveTabView extends React.PureComponent<{|
+  +activeBrowsingContextID: BrowsingContextID | null,
+  +showTabOnly: BrowsingContextID | null,
+  +changeShowTabOnly: typeof changeShowTabOnly,
+|}> {
+  _toggleShowTabOnly = () => {
+    const {
+      showTabOnly,
+      changeShowTabOnly,
+      activeBrowsingContextID,
+    } = this.props;
+    if (showTabOnly === null) {
+      changeShowTabOnly(activeBrowsingContextID);
+    } else {
+      changeShowTabOnly(null);
+    }
+  };
+
+  render() {
+    const { activeBrowsingContextID, showTabOnly } = this.props;
+    if (activeBrowsingContextID === null) {
+      return null;
+    }
+
+    return (
+      <form>
```

Comment:
Well, it doesn't do anything :) Wanted to keep a form element there because other elements also had their `form` elements, and thought it would be good to keep the same convention. But I can remove it